DigitalLearningSolutions.Web.Tests/Controllers/LearningPortal/RecommendedLearningControllerTests.cs

```diff
@@ -3,7 +3,9 @@
     using System.Collections.Generic;
     using System.Threading.Tasks;
     using DigitalLearningSolutions.Data.Models.External.Filtered;
+    using DigitalLearningSolutions.Data.Models.SearchSortFilterPaginate;

```

Comment:
This isn't being used from what I can see, same applies to [EditRolesViewModelTests.cs](https://github.com/TechnologyEnhancedLearning/DLSV2/pull/1055/files#diff-d4861828379bb4a5d76fd892e572c466ae8dac93368321c377f8306715e658ad) and [ViewDelegateControllerTests.cs](https://github.com/TechnologyEnhancedLearning/DLSV2/pull/1055/files#diff-103aebc1b2f338ea3cd331bef0597f7957ca6a2704b63520e36768dbceb0c6fd)